subjects/java/projects/travel-plan/README.md

```diff
@@ -0,0 +1,64 @@
+## Travel-Plan
+
+### Objectives
+
+The primary goal of this project is to construct a robust and scalable environment for the Travel Management System and to develop a comprehensive Admin Dashboard that serves as the control center for travel management operations.
+
+### Instructions
+
+This is the first part of the last project and for this part the students will be asked to build the working enviroment and impliment the admin part to ease the last part for the project.
+
+#### 1. Environment Configuration
+
+##### Configure a Microservices Architecture:
+-    Follow industry best practices for scalability and high availability.
+-    Implement multiple replicas of each microservice for load balancing and failover mechanisms.
+
+##### Database Installation and Configuration:
+-    Set up PostgreSQL and Neo4j databases.
+-    Ensure databases are containerized to facilitate scalability and replication.
+
+##### Continuous Integration/Continuous Delivery (CI/CD) Pipeline:
+-   Employ Jenkins for CI/CD and unit testing.
+-   Employ SonarQube for automating code quality checks
+
+##### Use of Docker and Ansible:
+-    Docker to support automated provisioning and deployment of microservices.
+-    Ansible for creating playbooks to deploy all system elements consistently and scalably.
+
+##### Outcome:
+-    Achieve an automated, scalable infrastructure.
+-    Prepare the groundwork for Admin Dashboard development and additional features of the Travel Management System.*
+
+#### 2. Development and Design
+
+- Craft an Admin Dashboard enabling administrators to oversee users, travels, and payment gateways.
+- the admin should be able to add, edit or delete any user, travel, payment method and all the related part correctly (cascading!!!) 
+- Ensure each travel entry includes a destination or trajectory, dates, duration, activities or itinerary, accommodation, and transportation details.
```

Comment:
What do you mean by `trajectory`? Should this be an array of coordination of point A and point B
What do you mean by `itinerary`? 

To make this easier to "consume" you could split this bullet point in a sub list with some extra details about each field. For example:
```
- Ensure each travel entry includes:
  - a destination,  in form text (or should this be a coordinate?), or a trajectory (how should this information be saved?)
  - dates (should this be an array of date-times or dates? What is this supposed to look like?)
  - duration, which will be a float reporting the total amount of hours (days/weeks ??) spent in the travel
  - activities ?? or itinerary ?? 
  - accommodation ... 
  - transportation details ...
```

Reply:
yes trajectory mean array of coordination instead of single destination

Reply:
if i give them a lot of details about the fields and table it's like I'm giving them a database diagram but i want them to think and discover what they have to do from the least information

Reply:
I feel implementing a database storing `destination` vs `trajectory` as an array of coordination is quite different in terms of complexity. 

Since this is going to be a part of a bigger project, it would be nice for the student to know if they're going to use the `trajectory` for the future steps of this project or if `destination` would be enough